internal/hcsoci/layers.go

```diff
@@ -262,13 +262,13 @@ func UnmountContainerLayers(layerFolders []string, guestRoot string, uvm *uvm.Ut
 
 	// Unload the storage filter followed by the SCSI scratch
 	if (op & UnmountOperationSCSI) == UnmountOperationSCSI {
-		containerScratchPathInUVM := ospath.Join(uvm.OS(), guestRoot, scratchPath)
-		logrus.Debugf("hcsshim::unmountContainerLayers CombinedLayers %s", containerScratchPathInUVM)
+		containerRoofFSPathInUVM := ospath.Join(uvm.OS(), guestRoot, rootfsPath)
```

Comment:
RoofFSPath :)

Reply:
It's a predefined constant used in a few places :)